### PR TITLE
App structure

### PR DESCRIPTION
**Pull request by rbhatt1999**
In this pull request, I've made the following changes:

- Selected the `financial Modelling API`.
- Built file structure for the React app.
- Prepared routes and navigation in the app
- Called data from the API
- Created tests for the application
- Styled the components
- Fixed Linter errors

### CODE REVIEW

File: src/Components/Pages/dashboard.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+.brand {
+  margin-left: 40px;
+}
+
+.stocklist {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  color: white;
+  font-size: 17px;
+}
```
Comment (rloterh) [inline src/Components/Pages/dashboard.module.css]: Your app works great! However, the styling of the list on your _Homepage_ does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that:
- [x] two cards per row are displayed for mobile screen, as seen in the template design.
- [ ] the cards of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (or slightly alternating color shade) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
Home Page | Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Rohit, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦   🏃 

## Highlights 🎯   
- Nice presentation of this project
- Good job acknowledging Nelson Sakwa on Behance for the [Original design idea](https://www.behance.net/sakwadesignstudio).
- API fetch properly implemented
- State management with Redux properly implemented!
- SPA feature working as intended!
- Properly implemented unit and integration test!
- Filter works great!  
- Professional ReadMe!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Pages/dashboard.module.css (hunk 1 above)
Comment (rbhatt1999) [reply src/Components/Pages/dashboard.module.css]: Behance blue theme dispatched

File: src/Components/Pages/dashboard.module.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  background: rgb(62, 97, 163);
+  padding: 10px;
+  width: 50%;
+}
+
+.card:nth-child(2n) {
+  background-color: rgb(67, 105, 176);
+}
+
+.card:first-child {
```
Comment (t-yanick) [inline src/Components/Pages/dashboard.module.css]: - [x] Great job working on this this project and implementing the changes requested by the previous reviewer. However you have missed something which we have to fix. You have selected the blue theme from Behance which is very ok, but as the previous reviewer indicated, your cards color should follow an **alternating dual-color pattern**. You currently have **dual-color**, but it is not alternating. I will present your screenshot and that from Behance below for clarity.


| Your design | Original design |
| -------------  | ----------------- |
| Screen Shot 2023-02-07 at 10 08 08 PM | Screen Shot 2023-02-07 at 10 00 54 PM         |
Take a careful look at both screenshots, you will observe that in the original design the color on each column are not the same for the cells of that column. But from your own screenshot, you have a single color on the left column and a single color on the right colum. The colors should alternate for the cards of the same column.

File: src/Components/Pages/dashboard.module.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+.brand {
+  margin-left: 40px;
+}
+
+.brand > h1 {
+  font-size: 20px;
+}
+
+.stocklist {
```
Comment (t-yanick) [inline src/Components/Pages/dashboard.module.css:29]: - [x] You have done well to make two columns for the cards in mobile view, but the very first card take the entire row. Kindly fix this issue. `Tesla, Inc` should not take the first row alone.
**See screenshot below**
Screen Shot 2023-02-07 at 10 08 08 PM

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @rbhatt1999 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @rbhatt1999 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**